x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/new_features_tour.tsx

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
The tour is now extracted to its own file and can be deleted in a follow-up PR in `main` later.